hedera-node/src/main/java/com/hedera/services/state/enums/ContractActionType.java

```diff
@@ -0,0 +1,31 @@
+package com.hedera.services.state.enums;
+
```

Comment:
License reminder

Reply:
Should be fixed now after applying spotless.